script.js

```diff
@@ -36,7 +36,7 @@ colorApp.getUserChoice = function() {
     const moodButton = document.querySelectorAll('.moods');
     moodButton.forEach( (button) => {
         button.addEventListener('click', function (event) {
-            const hueChoice = `/api/random/${event.target.value}`;
+            const hueChoice = `/scheme`;
             const buttonValue = event.target.value;
             colorApp.getColors(hueChoice);
```

Comment:
```suggestion
            colorApp.getColors(buttonValue);
```
Can pass buttonValue in to getColors() since hueChoice is redundant (see previous 2 comments)